src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -863,7 +872,7 @@
   <data name="ImageResizer_FilenameFormatHeader.Header" xml:space="preserve">
     <value>Filename format</value>
   </data>
-  <data name="ImageResizer_UseOriginalDate.Content" xml:space="preserve">
+  <data name="ImageResizer_UseOriginalDate.Header" xml:space="preserve">
     <value>Use original date modified</value>
```

Comment:
```suggestion
    <value>Use original date modified in file name</value>
```

Because the section header is only `File` this makes the setting a bit clearer. Maybe we should instead add an subtitle here like `Replace date variable with date modified instead of original date in file names.`

Reply:
![image](https://user-images.githubusercontent.com/9866362/133929718-0cc5a15c-8d24-41f0-abbe-2a8ce103da78.png)